src/mcp_analyzer/cli.py

```
@@ -348,76 +406,60 @@ async def _run_analysis(
-            tools = await client.get_tools()
+        if is_npx:
+            with console.status("[bold green]Launching NPX server..."):
+                server_info = await client.get_server_info()
+                tools = await client.get_tools()
 
-        actual_url = target
+            actual_url = client.get_server_url()
+            console.print(f"✅ NPX server launched at [cyan]{actual_url}[/cyan]")
+        else:
+            with console.status("[bold green]Connecting to MCP server..."):
+                server_info = await client.get_server_info()
```

Ruff (0.13.2)

Local variable `server_info` is assigned to but never used

Remove assignment to unused variable `server_info`

(F841)